src/main/resources/reference.conf

```diff
@@ -24,12 +24,12 @@ resourceTypes =
     },
     {
       name = "billing-project"
-      actionPatterns = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "launch_notebook", "alter_google_role", "share_policy::.+", "read_policy::.+"]
+      actionPatterns = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "list_notebook_cluster", "launch_notebook_cluster", "sync_notebooks_cluster", "delete_notebooks_cluster", "alter_google_role", "share_policy::.+", "read_policy::.+"]
```

Comment:
Not sure whether I prefer "notebook" or "notebooks" but they should all match

Reply:
good catch :)